about/join.qmd

```diff
@@ -17,13 +24,53 @@ The R Consortium is the mechanism for corporate entities and other organizations
 
 ## How R Consortium membership helps support the R Community
 
--   Supports the operations of the critical infrastructure that sustains the R ecosystem, anticipating challenges and planning for contingencies.  
--   Funds open-source community-driven projects that widely impact the R Community
--   Provides opportunities to form working relationships that transcend company affiliations.
--   Enables better identification and recruitment of data scientists working in R.
--   Convenes thought leadership events, summits, and panels among subject matter experts to discuss technology, standards, specifications, best practices, and collaboration
+```{r}
+suppressPackageStartupMessages({
+  library(dplyr)
+  library(gt)
+})
+
+df <- data.frame(
+  Benefits = c("One seat on the Board of Directors with full voting rights", "One seat on the ISC with full voting rights", "Elect Silver representatives to the Board and ISC ( 1 Board seat per 3 Silver members, 1 ISC seat for all Silver members )", "Company logo on R Consortium website and collateral", "R Consortium logo on company website", "Membership dues ( annually )"),
```

Comment:
Having spaces inside the parentheses looks appealing, but it means the trailing parens can be moved to a new line

![Screen Shot 2024-09-16 at 6 49 55 PM](https://github.com/user-attachments/assets/e700d7ad-9bf5-4a38-91df-85944e1c9e69)